pages.pt-BR/common/ab.md

```diff
@@ -0,0 +1,20 @@
+# ab
+
+> Ferramenta de benchmarking do servidor Apache. A ferramenta mais simples para realizar testes de carga.
```

Comment:
> A ferramenta mais simples para realizar testes de carga.

This is subjective, and borders on ad-like. I'd suggest removing it.

Reply:
I see this was translated literally from the English page, you're right it's subjective, it should be removed from there too. Perhaps something like "a simple tool for load testing", instead of "the simplest" would be ok, what do you think?

Reply:
@mebeim totally agree with that. Writing it that way almost look like an ad.

Reply:
I don't see why we need two sentences saying "a tool to do X". If we want to keep "testes de carga", then we could combine the sentences, e.g. something like this:

> Ferramenta do servidor Apache para realizar benchmarking e testes de carga.

What do you guys think?

Reply:
You're are right guys. 

What do you think about adding a section of CONTRIBUTING.md showing some good and bad examples about descriptions, translations, etc?

Reply:
@waldyrious you're right, that's repetitive. We should go with something like:

> Apache Benchmarking tool to perform load testing.

@andrik that could be a good idea, I wouldn't know where precisely to add it though. I'll take a closer look when I've got time.

Reply:
I think such a section would make more sense in the [style guide](https://github.com/tldr-pages/tldr/blob/master/contributing-guides/style-guide.md), but overall it's a good idea, yes.

Reply:
> Apache Benchmarking tool

@mebeim that got me thinking -- is this a tool of the Apache HTTP server, or a tool under the Apache organization's umbrella? If it's the latter, the Portuguese description needs to be changed, because it refers to the Apache server.

Reply:
@waldyrious it's developed by Apache, but it's general purpose, for *any* web server.

Reply:
Even though you can use ab on other web servers, it's distributed [within](https://httpd.apache.org/docs/2.4/programs/) Apache Web Server 

As we can't install it as a standalone program I think it would be better described as Apache Web Server Tool. What do you think?

Reply:
@andrik it is not really part of the Apache Web Server, it is part of `apache2-utils`, which includes [various utilities](https://packages.debian.org/stretch/apache2-utils). Anyway, the appropriate description IMHO would still be:

> Apache Benchmarking tool to perform load testing.

Or, maybe, more detailed:

> Apache Benchmarking tool to perform web server load testing.

Reply:
Hi @mebeim, you are right

I was fooled by the first sentence of [docs](https://httpd.apache.org/docs/2.4/programs/) of Apache :man_facepalming: 

> This page documents all the executable programs included with the Apache HTTP Server.

Reply:
> @andrik it is not really part of the Apache Web Server, it is part of `apache2-utils`, which includes [various utilities](https://packages.debian.org/stretch/apache2-utils), and the package description says `Provides some add-on programs useful for any web server.`.
> 
> Anyway, the appropriate description IMHO would still be:
> 
> > Apache Benchmarking tool to perform load testing.
> 
> Or, maybe, more detailed:
> 
> > Apache Benchmarking tool to perform web server load testing.

I think the latter is good, does everybody agree?

Reply:
> I think the latter is good, does everybody agree?

I agree 👍

Reply:
@andrik I agree too :+1: